aws-terraform/ecs/task-definitions/scim.json

```diff
@@ -0,0 +1,59 @@
+[
+    {
+      "name": "scim-bridge",
+      "image": "1password/scim:v1.6.1",
+      "cpu": 128,
+      "memory": 512,
+      "essential": true,
+      "dependsOn": [
+        {
+          "containerName": "redis",
+          "condition": "START"
+        }
+      ],
+      "portMappings": [
+        {
+          "containerPort": 3002,
+          "hostPort": 3002
+        }
+      ],
+      "entrypoint": ["/op-scim/op-scim", "--redis-host=localhost"],
+      "environment": [
+        { "name": "OP_REDIS_URL", "value": "localhost" },
```

Comment:
Yep. For clarity here, you only need one of `OP_REDIS_URL` or `--redis-host=localhost`. You don't need both.

My personal preference among all the example deployments is to use environment variables (i.e: `OP_REDIS_URL`) and to not use command arguments (i.e: `--redis-host`) since the former is much easier to manage from a configuration standpoint.

Reply:
Unfortunately, I tried removing the --redis-host=localhost and that breaks it, but my preference would be the same, otherwise.

Reply:
I believe the variable would be `OP_REDIS_HOST` not `x_URL`